policy-suggest-dns.yaml

```diff
@@ -0,0 +1,62 @@
+APIVersion: 1
+label: recipe:policy-suggest:dns
+data:
+  recipes:
+    - name: Create DNS rule
+      description: Creates a basic DNS rule for the current namespace.
+      label: recipe:policy-suggest:dns
+      metadata:
+        - "@aporeto:author=aporeto"
+      associatedTags:
+        - "aporeto:recipe:placement=policy-suggest"
+        - "aporeto:recipe:identity=network-service-rules"
+      deploymentMode: NamespaceUnique
+      targetIdentities:
+        - externalnetwork
+        - networkrulesetpolicy
+      propagate: true
+      longDescription: |-
+        ### Create a DNS rule
+
+        This recipe will create an External Network and Network Ruleset Policy to
+        enable access to DNS port UDP/53 on the current namespace.
+
+      template: |-
+        {{`
+        APIVersion: 1
+        data:
+          externalnetworks:
+          - name: dns
+            protected: true
+            entries:
+            - {{ .Values.network | quote }}
+            associatedTags:
+            - 'externalnetwork:name=dns'
+          networkrulesetpolicies:
+          - name: dns
+            protected: true
+            outgoingRules:
+            - action: Allow
+              object:
+              - - 'externalnetwork:name=dns'
+              protocolPorts:
+              - udp/53
+            subject:
+            - {{- range $index, $orgmeta := .Aporeto.OrganizationalMetadata }}
+              {{- if hasPrefix "@org:" $orgmeta }}
```

Comment:
I think we have a check to validate all organizational metadata are starting with `@org:` no?

Reply:
But it does not hurt to double check 👍